authors/models.py

```diff
@@ -1,3 +1,22 @@
+import uuid
 from django.db import models
+from django.contrib.auth.models import User
 
 # Create your models here.
+class Author(models.Model):
+    id = models.UUIDField(primary_key=True, editable=False, default=uuid.uuid4)
+    user = models.OneToOneField(User, on_delete=models.CASCADE) # one2one with django user
+    friends = models.ManyToManyField("self") # bidirectional/symmetrical by default
+
+    displayName = models.CharField(max_length=30) # maximum 30 chars for display name
```

Comment:
which variable naming convention are we following? camelCase or snake_case? 

Everything else lgtm

Reply:
good catch. let's use snake_case